### DISCUSSION TITLE
installation problem

### DISCUSSION THREAD

**Discussion by gniloo** [TomographicImaging/CIL-Demos#129] [Q&A]
when I want to install CIL toolbox with this code (conda create --name cil -c conda-forge -c intel -c ccpi cil=23.0.1 astra-toolbox tigre ccpi-regulariser tomophantom "ipywidgets<8")
this gives this error ( A directory already exists at the target location 'C:\Users\LENOVO\anaconda3\envs\cil'
but it is not a conda environment.)
would you please help me to solve this problem.

**Comment by paskino** [comment]
well, it looks like you already have a directory named `cil` in your anaconda installation. The easiest solution is to change the environment name

```
conda create --name cil_latest -c conda-forge -c intel -c ccpi cil=23.0.1 astra-toolbox tigre ccpi-regulariser tomophantom "ipywidgets<8"
```

The other option is to permanently delete it, `C:\Users\LENOVO\anaconda3\envs\cil`. Although I would say that this is rather safe, before doing so, make sure it does not contain files you'll regret deleting.
